src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import 'material-icons';
+import styled from 'styled-components';
+import Avatar from '@material-ui/core/Avatar';
+import SearchBar from '../SearchBar';
+import ThemeSwitch from '../ThemeSwitch/ThemeSwitch.component';
+
+const HeaderClass = styled.header`
```

Comment:
Using the name Class in a component is a little misleading. I get you are referring to CSS class selector. Remember that you are working with JS too, and classes in that context refer to stateful components`StyledHeader` could be a little bit more descriptive of the intent behind this component.

Though, I liked how you didn't export this one since it's just a styled container. Just think about it.